js/foundation.magellan.js

```diff
@@ -238,6 +240,13 @@ Magellan.defaults = {
    * @default 'is-active'
    */
   activeClass: 'is-active',
+    /**
+  * If >= 0 index of parents of active link, where activeClass is added/removed
+  * @option
+  * @type {number}
+  * @default -1
+  */
+ activeParents: -1,
   /**
```

Comment:
If you use jQuery's `eq()` as suggested above, this has to be `null`default and e.g. `0` if you want the first parent.